packages/auth/src/__tests__/buildLegacyAccessPolicy.test.js

```diff
@@ -104,28 +107,43 @@ describe('buildLegacyAccessPolicy', () => {
       accessPolicy = await buildLegacyAccessPolicy(models, user.id);
     });
 
-    it('should have Demo Land public access, as interpreted by legacy parser', () => {
-      expect(hasAccess(accessPolicy, 'surveys', ['DL'], 'Public')).to.equal(true);
-      expect(hasAccess(accessPolicy, 'surveys', ['DL'], 'Donor')).to.equal(false);
-      expect(hasAccess(accessPolicy, 'surveys', ['DL'], 'Admin')).to.equal(false);
-    });
-
-    it('should have Tonga admin, donor, and public access, as interpreted by legacy parser', () => {
-      expect(hasAccess(accessPolicy, 'surveys', ['TO'], 'Public')).to.equal(true);
-      expect(hasAccess(accessPolicy, 'surveys', ['TO'], 'Donor')).to.equal(true);
-      expect(hasAccess(accessPolicy, 'surveys', ['TO'], 'Admin')).to.equal(true);
+    const testData = [
+      [
+        'should have Demo Land public access, as interpreted by legacy parser',
+        [
+          [['surveys', ['DL'], 'Public'], true],
```

Comment:
> 'surveys' is common in all test cases here, so it could be explicitly specified when `hasAccess` is called in line 131
> 
> Similarly for the other test data here

Good idea!